rgf/utils.py

```diff
@@ -570,86 +568,66 @@ def predict_proba(self, X):
                 y = softmax(y)
         return y
 
-    def predict(self, X):
-        """
-        Predict class for X.
 
-        The predicted class of an input sample is computed.
+class RGFRegressorMixin(object):
```

Comment:
Add `_get_sample_weight()`, `_fit_regression_task()` and `_check_n_features()` stubs with `NotImplementedError` to `RGFRegressorMixin`